spec/requests/host_aggregates_spec.rb

```diff
@@ -1,4 +1,5 @@
 RSpec.describe 'HostAggregates API' do
+  include Spec::Support::SupportsHelper
   describe 'GET /api/host_aggregates' do
```

Comment:
Minor but a newline before this would look better IMO

Reply:
Oh sorry I meant a newline before _this_ line aka after the `include` also rubocop is warning about the newline before the `include` now